crypto/ec/ec_key.c

```diff
@@ -119,9 +119,11 @@ EC_KEY *EC_KEY_copy(EC_KEY *dest, const EC_KEY *src)
                     return NULL;
             }
             if (!BN_copy(dest->priv_key, src->priv_key))
+                BN_free(dest->priv_key);
```

Comment:
This change is incorrect and will result in a double free. After this change `dest->priv_key` is still set so when `dest` eventually gets freed it will attempt to free `dest->priv_key` again.